.github/workflows/upload_image.yml

```diff
@@ -16,17 +16,31 @@ jobs:
     outputs:
       image_tag: ${{ steps.image_tag.outputs.image_tag }}
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
+        with:
+          # it is required for fetching branches which contains certain tag
+          fetch-depth: 0
 
-      - name: Extract Image Tag
+      - name: Extract Image Tags
         shell: bash
         run: |
           IMAGE_TAG=${GITHUB_REF##*/}
```

Comment:
This can be replaced by `${{ github.ref_name }}`.

Reply:
Updated! PTAL